microsoftgraph/microsoft-graph-explorer-v4
# Fix: Persist snippet tabs

## Overview
Fixes #1948 

### Demo
Normal snippet tab
<img width="633" alt="image" src="https://user-images.githubusercontent.com/45680252/182102292-17b98560-e0e6-4777-baa9-69b650182ee3.png">

Modal snippet tab
<img width="713" alt="image" src="https://user-images.githubusercontent.com/45680252/182102362-4f37fa59-bdeb-4e9f-bd4a-471dc1ea2afb.png">

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* How to test this PR
Check out this branch and run it locally
Click on the Code snippets tab and select Powershell
Click on Expand tab
Notice that the snippet tab on the opened modal is still Powershell
Now close the Modal
Notice that the snippet is still displayed
